.circleci/config.yml

```diff
@@ -40,6 +40,14 @@ aliases:
       - MYSQL_USER=test
       - MYSQL_DATABASE=test
 
+  - &IMAGE_DOCKER_MONGODB
+    image: bitnami/mongodb:4.0
```

Comment:
Any specific reason why we are not using `image: circleci/mongo:4.0`? https://circleci.com/docs/2.0/circleci-images/#mongodb